django-stubs/contrib/messages/storage/cookie.pyi

```diff
@@ -14,7 +14,20 @@ class MessageEncoder(json.JSONEncoder):
 class MessageDecoder(json.JSONDecoder):
     def process_messages(self, obj: Any) -> Any: ...
 
+class MessagePartSerializer:
+    def dumps(self, obj: Any) -> Sequence[str]: ...
+
+class MessagePartGatherSerializer:
+    def dumps(self, obj: Any) -> bytes: ...
+
+class MessageSerializer:
+    def loads(self, data: bytes | bytearray) -> Any: ...
+
 class CookieStorage(BaseStorage):
     cookie_name: str
     max_cookie_size: int
     not_finished: str
+    not_finished_json: str
+
+def bisect_keep_left(a: list[int], fn: Callable[[list[int]], bool]) -> int: ...
```

Comment:
I guess it is fine not to bother with `Sequence` or anything in such a utility function.

Reply:
Thank you :)